server/app/views/applicant/ApplicantBaseFragment.html

```diff
@@ -0,0 +1,24 @@
+<head th:fragment="pageHeader">
```

Comment:
This file will probably split into more than one as time goes on. For now it's just a place for fragments that are shared across multiple pages, but the footer at least will grow into its own component at some point, and there will be more fragments like this